IPython/terminal/interactiveshell.py

```diff
@@ -69,26 +87,24 @@ def get_default_editor():
 
     if os.name == 'posix':
         return 'vi'  # the only one guaranteed to be there!
-    else:
-        return 'notepad' # same in Windows!
+    return 'notepad' # same in Windows!
```

Comment:
That's typically a change that's generally considered good, though this function is relatively simple, and that will make any further backport or history digging around this file more difficult. I dont' think it is really necessary.